builder/parser/line_parsers.go

```diff
@@ -10,13 +10,12 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strconv"
 	"strings"
 	"unicode"
 )
 
 var (
-	errDockerfileJSONNesting = errors.New("You may not nest arrays in Dockerfile statements.")
+	errDockerfileNotStringArray = errors.New("When using JSON array syntax, arrays must be comprised of strings only.")
```

Comment:
This phrase is only a 6.8 on the Flesch-Kincaid Grade Level for
Readability. ;)

https://github.com/docker/docker/blob/master/docs/sources/contributing/docs_style-guide.md#prose-style